bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en-features.html

```diff
@@ -0,0 +1,118 @@
+    <p>
+      <a class="mzp-c-button mzp-t-secondary js-open-about-protections" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}"  data-cta-text="View my protections" data-cta-type="button">
+        View my protections
+      </a>
+    </p>
+  {% endcall %}
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md',
+    body_class='mzp-l-split-v-center',
+    media_class='mzp-l-split-h-center',
+    image=resp_img('img/firefox/whatsnew/whatsnew109-en/addon.jpg',
+      srcset={ 'img/firefox/whatsnew/whatsnew109-en/addon-high-res.jpg': '2x' },
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+  ) %}
+    <h2>Organize all your open tabs with a magic button</h2>
+    <p>Add OneTab to your browser — and with one click, suddenly all your open tabs become a clean list.</p>
+    <p>
+      <a class="mzp-c-button mzp-t-secondary" href="https://addons.mozilla.org/firefox/addon/onetab/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}"  data-cta-text="Learn" data-cta-type="button">
+        Learn more
+      </a>
+    </p>
+  {% endcall %}
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed',
+    body_class='mzp-l-split-v-center',
+    media_class='mzp-l-split-h-center',
+    image=resp_img('img/firefox/whatsnew/whatsnew109-en/pdf.jpg',
+      srcset={ 'img/firefox/whatsnew/whatsnew109-en/pdf-high-res.jpg': '2x' },
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+  ) %}
+    <h2>Edit PDFs directly in Firefox</h2>
+    <p>No need to print out medical forms, financial documents or contracts — edit and sign any PDF right in Firefox and get back to the fun stuff.</p>
+    <p>
+      <a class="mzp-c-button mzp-t-secondary" href="https://assets.mozilla.net/wnp109-eu/wnp-fx109-uk.pdf" data-cta-text="Try it now" data-cta-type="button">
+        Try it now
+      </a>
+    </p>
+  {% endcall %}
+
+  <div class="c-footer-heading mzp-t-dark">
+    <div class=" mzp-l-content mzp-t-content-xl">
+      <h2>Discover other products from Mozilla</h2>
+    </div>
+  </div>
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-t-dark',
+    body_class='mzp-l-split-v-center',
+    media_class='mzp-l-split-h-center',
+    image=resp_img('img/firefox/whatsnew/whatsnew109-en/privacy-header.jpg',
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+  ) %}
+    <div class="mzp-c-wordmark mzp-t-wordmark-md mzp-t-product-vpn"></div>
+    <h3>Protect what’s important all year long</h3>
+    <p>Protect your location & activity using Mozilla VPN.</p>
+    <p>With the flip of a switch, Mozilla VPN works in the background to protect your entire device.</p>
+    <p>
```

Comment:
**Nit (accessibility/blocking):** When focusing on this button, both text and background colors are light colors , and their color contrast ratios are not accessible to readers.

<img width="547" alt="Screen Shot 2023-01-12 at 5 40 23 PM" src="https://user-images.githubusercontent.com/42309026/212196691-4aec025c-8445-485b-8311-37fc403ef6cf.png">

Reply:
This is a bug in Protocol. Filed https://github.com/mozilla/protocol/issues/842 to fix it upstream but I'll add a temp fix here as well.